packages/management/addons-manager-v2/bundle/config/overlays/overlay-addons-manager.yaml

```diff
@@ -1,57 +0,0 @@
-#@ load("@ytt:data", "data")
-#@ load("@ytt:overlay", "overlay")
-
-#@ if data.values.tanzuAddonsManager.createNamespace:
----
-apiVersion: v1
-kind: Namespace
-metadata:
-  name: #@ data.values.tanzuAddonsManager.namespace
-#@ end
-
-#@overlay/match by=overlay.subset({"kind":"ServiceAccount", "metadata": {"name": "tanzu-addons-manager-sa"}})
----
-apiVersion: v1
-kind: ServiceAccount
-metadata:
-  namespace: #@ data.values.tanzuAddonsManager.namespace
-
-#@overlay/match by=overlay.subset({"kind":"ClusterRoleBinding", "metadata": {"name": "tanzu-addons-manager-clusterrolebinding"}})
----
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRoleBinding
-subjects:
-#@overlay/match by=overlay.subset({"kind":"ServiceAccount", "name": "tanzu-addons-manager-sa"})
-- kind: ServiceAccount
-  name: tanzu-addons-manager-sa
-  namespace: #@ data.values.tanzuAddonsManager.namespace
-
-#@overlay/match by=overlay.subset({"kind":"Deployment", "metadata": {"name": "tanzu-addons-controller-manager"}})
----
-apiVersion: apps/v1
-kind: Deployment
```

Comment:
question: is the reason of removing this file because all overlay logic/template has been handled by `upstrea/addons-manager.yaml` ?